frontend/awx/resources/inventories/inventoryGroup/GroupPageAdd.cy.tsx

```diff
@@ -0,0 +1,38 @@
+import { GroupPageAdd } from './GroupPageAdd';
```

Comment:
We should probably have some RBAC tests as part of this PR also.  

- We should test that the the Create group button on the list is disabled.  
- We should also have some e2e tests that assert that we can get from login, to the form, create a group, and then delete the group.  (Consider reaching out to QE for their thoughts on other e2e tests).